metrilo_open_api_specification.yml

```diff
@@ -318,6 +372,48 @@ components:
                   example:
```

Comment:
`Tag labels that are assigned to customers in the Metrilo application. (tags are merged, not overwritten)`